sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.aest.controller.reverseScholasticStanding.e2e-spec.ts

```diff
@@ -44,14 +46,29 @@ describe("StudentScholasticStandingsAESTController(e2e)-reverseScholasticStandin
     sharedUser = await db.user.save(createFakeUser());
   });
 
+  beforeEach(async () => {
+    await db.notification.update(
+      {
+        dateSent: IsNull(),
+        notificationMessage: {
+          id: NotificationMessageType.ScholasticStandingReversalNotification,
+        },
+      },
+      {
+        dateSent: new Date(),
+      },
+    );
+  });
+
   afterEach(async () => {
     // Reset the current date mock.
     MockDate.reset();
   });
 
   it(
     "Should reverse an active scholastic standing change and create a student note" +
-      ` when the scholastic standing change type is ${StudentScholasticStandingChangeType.StudentDidNotCompleteProgram}.`,
+      ` when the scholastic standing change type is ${StudentScholasticStandingChangeType.StudentDidNotCompleteProgram}` +
+      " and send an email notification to the student for the same.",
```

Comment:
Minor. The part `send an email notification to the student for the same` must be a part of Should and not after when as per the test description pattern we follow.